src/handlers/comment/handlers/query.ts

```diff
@@ -19,7 +19,7 @@ export const query = async (body: string) => {
     return `Skipping '/query' because of no issue instance`;
   }
 
-  const regex = /^\/query\s@(\w+)$/;
+  const regex = /(?<=\/query @)\w+/g
```

Comment:
this regex is different than your debugger and the positive lookbehind seems rather unnecessary, this can be done with simple capturing groups.

also please make sure it still matches if you put multiple spaces between like `/query    @test`